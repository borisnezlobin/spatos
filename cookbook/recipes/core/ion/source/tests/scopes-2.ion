if test 1 == 1
  let x = 1

  fn print
    echo ${super::x}
    echo ${super::y}
  end

  if test 1 == 1
    let y = 2
    print
  end
end
