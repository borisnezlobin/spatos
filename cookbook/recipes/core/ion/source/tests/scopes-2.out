1
ion: expansion error: Variable "super::y" does not exist
